test/parallel/util-parse-env.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../../test/common/fixtures');
+const assert = require('node:assert');
+const util = require('node:util');
+const fs = require('node:fs');
+
+{
```

Comment:
Any reason the tests in this file don’t use the test runner? That would help to organize them.

Reply:
I didn't see any specific reason to use the test runner, since the tests are pretty straight forward, and aren't complex, or require isolation.